### ISSUE TITLE
Different loss functions

### ISSUE THREAD

**Issue by Wanguy** [Wanguy/Gaze_HybirdModel#3]
Try different loss functions and learning rates.
